docs/community/tutorials-contest/2022/avax-subnet-customization/README.md

```diff
@@ -0,0 +1,975 @@
+    }
+  }
+}
+```
+
+Now we're done! The final config looks like:
+
+```json
+{
+    "config": {
+        "chainId": 321123,
+        "homesteadBlock": 0,
+        "eip150Block": 0,
+        "eip150Hash": "0x2086799aeebeae135c246c65021c82b4e15a2c451340993aacfd2751886514f0",
+        "eip155Block": 0,
+        "eip158Block": 0,
+        "byzantiumBlock": 0,
+        "constantinopleBlock": 0,
+        "petersburgBlock": 0,
+        "istanbulBlock": 0,
+        "muirGlacierBlock": 0,
+        "SubnetEVMTimestamp": 0,
+        "feeConfig": {
+            "gasLimit": 5000000,
+            "targetBlockRate": 5,
+            "minBaseFee": 60000000000,
+            "targetGas": 10000000,
+            "baseFeeChangeDenominator": 50,
+            "minBlockGasCost": 0,
+            "maxBlockGasCost": 5000000,
+            "blockGasCostStep": 10000
+        },
+        "contractDeployerAllowListConfig": {
+            "blockTimestamp": 0,
+            "adminAddresses": [
+                "0x0000000b9af48743ef1188f3F20c9b8B90F52a5b"
+            ]
+        },
+        "contractNativeMinterConfig": {
+            "blockTimestamp": 0,
+            "adminAddresses": [
+                "0x0000000b9af48743ef1188f3F20c9b8B90F52a5b"
+            ]
+        }
+    },
+    "alloc": {
+        "0000000b9af48743ef1188f3F20c9b8B90F52a5b": {
+            "balance": "0x3635C9ADC5DEA00000"
+        }
+    },
+    "nonce": "0x0",
+    "timestamp": "0x0",
+    "extraData": "0x00",
+    "gasLimit": "5000000",
+    "difficulty": "0x0",
+    "mixHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+    "coinbase": "0x0000000000000000000000000000000000000000",
+    "number": "0x0",
+    "gasUsed": "0x0",
+    "parentHash": "0x0000000000000000000000000000000000000000000000000000000000000000"
+}
+```
+
+# Creating and Deploying the Subnet Using the Custom Genesis File
+
+We will use `genesis.json` to create a new Subnet. This part is pretty simple.
+
+Let's first delete the old Subnet that we created using the wizard.
+
+```bash
+$ avalanche Subnet delete testSubnet
+
+> Deleted Subnet
+```
+
+Next, we will create a new Subnet using the genesis file.
+
+```bash
+# avalanche Subnet create <SubnetName> --file <filepath>
+
+$ avalanche Subnet create game --file ./genesis.json
+# Use SubnetEVM
+
+> Using specified genesis
+> ✔ SubnetEVM
+> Successfully created genesis
+```
+
+Now let's deploy it!
+  
+```bash
+# avalanche Subnet deploy <SubnetName>
+$ avalanche Subnet deploy game
+
+# choose local network as deployment target
+
+> Metamask connection details (any node URL from above works):
+> RPC URL:          http://127.0.0.1:27200/ext/bc/HGwbjkQsvrqy2mncMWsA2YnrSWEybVmLcZJwz7nb5gNVvcPoi/rpc
+> Funded address:   0x0000000b9af48743ef1188f3F20c9b8B90F52a5b with 1000
+> Network name:     game
+> Chain ID:         321123
+> Currency Symbol:  TEST
+```
+
+Awesomeness! Great work!
+
+# Deploying the Game Contract
```

Comment:
Per [Markdownlint](https://github.com/DavidAnson/markdownlint) You shouldn't have multiple `<h1>`s in the same document.

> MD025/single-title/single-h1: Multiple top-level headings in the same document

[More info: MD025](https://github.com/DavidAnson/markdownlint/blob/v0.25.1/doc/Rules.md#md025---multiple-top-level-headings-in-the-same-document)